Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM gradle:jdk-alpine
+
+USER root
+
+RUN apk add bash
```

Comment:
Just wondering: what do you need bash for?

Reply:
The gradle install generates a bash script to run the analyzer. I'll probably write my own and remove the dependency on bash

Reply:
Ah okay, nice